dll/win32/shell32/folders/CDrivesFolder.cpp

```diff
@@ -409,7 +429,7 @@ getIconLocationForDrive(IShellFolder *psf, PCITEMID_CHILD pidl, UINT uFlags,
 }
 
 static HRESULT
-getLabelForDrive(LPWSTR wszPath, LPWSTR wszLabel)
+getLabelForDriveFromInf(PCWSTR wszPath, LPWSTR szLabel, UINT cchMax)
```

Comment:
```suggestion
getLabelForDriveFromAutoRun(PCWSTR wszPath, LPWSTR szLabel, UINT cchMax)
```
Or `getDriveLabelFromAutoRun` .  "from Inf" is a too vague (which inf file? ...)